### PR TITLE
Adjust labels and configuration

### PR DESCRIPTION
**Pull request by lpugin**
* Adjust configuration for works
* Add labels for works and work nodes

### CODE REVIEW

File: config/locales/marc_records/en.yml [hunk 1]
```diff
@@ -394,6 +394,7 @@ en:
     dedicatee: Dedicatee
     department: Department
     depositor: Depositor
+    deprecated_fields: Obsolete fields (do not use)
```
Comment (ahankinson) [inline config/locales/marc_records/en.yml:397]: These are from the translations files https://github.com/rism-digital/translations/blob/main/locales/en.yml

File: config/locales/marc_records/en.yml [hunk 2]
```diff
@@ -763,6 +764,7 @@ en:
     scoring_in_movement: Scoring in movement
     scoring_summary: Scoring summary
     scribe: Scribe
+    secondary_catalog_works: Secondary catalog of works
```
Comment (ahankinson) [inline config/locales/marc_records/en.yml:767]: Here too.

File: config/locales/marc_records/en.yml (hunk 1 above)
Comment (xhero) [reply config/locales/marc_records/en.yml:397]: Yes I will update the translations repo by hand when we have all the new ones

Comment (ahankinson) [reply config/locales/marc_records/en.yml:397]: ok

**Pull request merged**